.github/workflows/sync-examples.yml

```
@@ -1,12 +1,90 @@
 name: Sync example notebooks from main PGM repo to PGM-workshop
 
 on:
+  push:
   workflow_dispatch:
+  schedule:
+    - cron: "32 2 * * *" # Based on UTC time
```

```suggestion
    - cron: "0 2 * * *" # Based on UTC time
```

see also https://crontab.guru/#32_2_*_*_*